ops/cloud-deployment/backend-api-deploy.bicep

```diff
@@ -187,6 +208,20 @@ var applicationSettings = concat([
   ],
   !empty(databaseConnectionString) ? [ { name: 'SQL_SERVER_CONN_STRING', value: databaseConnectionString } ] : []
 )
+var ipSecurityRestrictionsRules = concat([ {
+      ipAddress: 'Any'
+      action: 'Deny'
+      priority: 2147483647
+      name: 'Deny all'
+      description: 'Deny all access'
+    } ],
+  allowVeracodeScan ? [ {
+      ipAddress: '3.32.105.199/32'
```

Comment:
should this be obfuscated?

Reply:
@governmentSponsored I'd be fine with that, but this is published on Veracode's documentation I believe.